core/templates/dev/head/player/flag_exploration_popup.html

```diff
@@ -0,0 +1,42 @@
+<script type="text/ng-template" id="modals/flagExploration">
+  <div class="modal-header">
+    <h3>
+      <span>Report Exploration</span>
+    </h3>
+  </div>
+  <div class="modal-body oppia-long-text">
```

Comment:
I've been thinking through this, and I think that we should _always_ ask for a description of what the issue is, otherwise it's not clear what to look at. So maybe do something like:

```
**What is the issue?**
- Misleading content, spam or ads
- Poor learning experience
- Other
```

and then, regardless of whatever they select, show the textarea:

```
Please provide additional details for the moderators:
[TEXTAREA]
```